spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfigurationTests.java

```diff
@@ -51,118 +42,46 @@
 import org.springframework.cloud.skipper.client.SkipperClient;
 import org.springframework.cloud.task.configuration.SimpleTaskAutoConfiguration;
 import org.springframework.cloud.task.repository.TaskRepository;
-import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 import org.springframework.context.annotation.Bean;
-import org.springframework.core.env.ConfigurableEnvironment;
-import org.springframework.core.env.MapPropertySource;
-import org.springframework.core.env.MutablePropertySources;
-import org.springframework.core.env.StandardEnvironment;
 import org.springframework.hateoas.config.EnableHypermediaSupport;
 import org.springframework.security.authentication.AuthenticationManager;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.mockito.Mockito.mock;
 
 /**
  * @author Glenn Renfro
  * @author Ilayaperumal Gopinathan
  * @author Gunnar Hillert
+ * @author Michael Wirth
  */
 public class DataFlowServerConfigurationTests {
 
-	private AnnotationConfigApplicationContext context;
-
-	private ConfigurableEnvironment environment;
-
-	private MutablePropertySources propertySources;
-
-	@Before
-	public void setup() {
-		context = new AnnotationConfigApplicationContext();
-		context.setId("testDataFlowConfig");
-		context.register(DataFlowServerConfigurationTests.TestConfiguration.class,
-				SecurityAutoConfiguration.class, DataFlowServerAutoConfiguration.class,
-				DataFlowControllerAutoConfiguration.class, DataSourceAutoConfiguration.class,
-				DataFlowServerConfiguration.class, PropertyPlaceholderAutoConfiguration.class,
-				RestTemplateAutoConfiguration.class, HibernateJpaAutoConfiguration.class, WebConfiguration.class,
-				SchedulerConfiguration.class, JacksonAutoConfiguration.class, SimpleTaskAutoConfiguration.class,
-				ResourceLoadingAutoConfiguration.class, ComposedTaskRunnerConfigurationProperties.class);
-		environment = new StandardEnvironment();
-		propertySources = environment.getPropertySources();
-	}
-
-	@After
-	public void teardown() {
-		if (context != null) {
-			context.close();
-		}
-	}
-
-	/**
-	 * Verify that embedded server starts if h2 url is specified with default properties.
-	 */
-	@Test
-	@Ignore
-	public void testStartEmbeddedH2Server() {
```

Comment:
It would be a good idea to add 1 test that starts up the H2 server and make sure all is well.

Reply:
I re-attached the `testStartEmbeddedH2Server` and the `testDoNotStartEmbeddedH2Server` again